src/window.rs

```diff
@@ -193,16 +193,25 @@ impl Window {
     ) -> Result<Window> {
         let event_loop = EventsLoop::new();
 
+        fn create_gl_window(
+            window: WindowBuilder,
+            event_loop: &EventsLoop,
+            srgb: bool,
+        ) -> ::std::result::Result<glutin::GlWindow, glutin::CreationError> {
+            let context = ContextBuilder::new()
+                .with_srgb(srgb)
+                .with_vsync(true);
+            ::glutin::GlWindow::new(window, context, event_loop)
+        }
```

Comment:
I'm not a huge fan of inner functions, I think a normal function would be better in this case and then just use `#[inline]`. But maybe @jwilm thinks differently about this.

Reply:
I would prefer to not have the function definition nested if that's what you're getting at. `#[inline]` shouldn't be necessary though since this isn't exactly performance sensitive.

Reply:
Yeah I was talking about the nesting. And adding `#[inline]` would probably not make any difference.